tmdb/tmdb.go

```diff
@@ -649,10 +649,17 @@ func (credits *Credits) GetWriters() []string {
 	return writers
 }
 
-func GetImageQualities() (posterQuality, fanArtQuality, logoQuality, thumbnailQuality string) {
-	posterQuality = ImageQualitiesPoster[config.Get().TMDBImagesQuality]
-	fanArtQuality = ImageQualitiesFanArt[config.Get().TMDBImagesQuality]
-	logoQuality = ImageQualitiesLogo[config.Get().TMDBImagesQuality]
-	thumbnailQuality = ImageQualitiesThumbnail[config.Get().TMDBImagesQuality]
+type ImageQualityBundle struct {
+	Poster    string
+	FanArt    string
+	Logo      string
+	Thumbnail string
+}
+
+func GetImageQualities() (imageQualities ImageQualityBundle) {
+	imageQualities.Poster = ImageQualitiesPoster[config.Get().TMDBImagesQuality]
```

Comment:
probably would be cleaner to use return+assign:

```go

return ImageQualityBundle{
   FanArt: ImageQualitiesFanArt[config.Get().TMDBImagesQuality],
   Logo: ImageQualitiesLogo[config.Get().TMDBImagesQuality],
   ....
}
```

Reply:
done